apps/ftu/index.html

```diff
@@ -19,7 +19,6 @@
   <link rel="stylesheet" href="style/style.css">
   <link rel="stylesheet" href="style/status.css">
   <link rel="stylesheet" href="style/overlay.css">
-  <link rel="stylesheet" href="style/fb_extensions.css">
   <link rel="stylesheet" href="style/tutorial.css">
```

Comment:
actually that was my mistake, we do need them, but renaming to avoid 'fb' connection